findanagram.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('lib/anagrams.rb')
+
+text_file, *input_word = ARGV
+
+input_word.each do |word|
+  anagram = Anagrams.new(text_file)
```

Comment:
@zzokki81 I explicitly said:

> An instance of the class should analyze the dictionary only once and then be able to perform all the required actions.

Why are you creating an object each time you want to get anagrams of a word? Also, you are not analyzing the dictionary only once.

This results in slow execution:

```
$ ruby findanagram.rb anagrams-wordlist.txt undo undo undo undo             *[feature/anagrams] 
Anagrams for given word 'undo' is: 
udon undo
0.881240999995498
Anagrams for given word 'undo' is: 
udon undo
0.8340949999983422
Anagrams for given word 'undo' is: 
udon undo
0.812352999986615
Anagrams for given word 'undo' is: 
udon undo
0.8182730000116862
```

For each word it takes just shy of a second to get anagrams - even if it's the same word.